web/vtadmin/tests/handlers.ts

```diff
@@ -0,0 +1,5 @@
+import { graphql, rest } from 'msw'
+
```

Comment:
We should have a header here

Reply:
Ah thank you! Totally forgot about those!